src/app/index.ts

```diff
@@ -259,10 +260,11 @@ module.exports = yo.extend({
         
         if (this.project.isExcelFunctionsProject)
         {
-          this.fs.copyTpl(this.templatePath(`excel-custom-functions-preview/**`), this.destinationPath(), templateFills, null, { globOptions: { ignore: `**/*.placeholder` }});
+          // copy over custom functions files from Excel Custom Functions repo (this is the model we would like to have for all projects).
+          git.Clone("https://github.com/OfficeDev/Excel-Custom-Functions", this.destinationPath());
+
+          // we can ultimatey create a repo with basefiles as well such as certs
           this.fs.copy(this.templatePath(`${language}/base/certs`), this.destinationPath('certs'), { globOptions: { ignore: `**/*.placeholder` }});

```

Comment:
Wouldn't we need the base stuff to be part of the git project? If so they wouldn't be copied here.

Reply:
I think it might be easier to have a separate base repo so we only have to update the base files (e,g, certs, asset files)  one time rather than having to do it numerous times in each and every project repo